visualisation_temporal.py

```diff
@@ -0,0 +1,118 @@
+# packages
+import h5py
+import json
+import os
+import pandas as pd
+import pickle
+import plotly.io as pio
+import plotnine as p9
+import numpy as np
+import random
+
+
+
+###################################################
+# USER INPUT: config to use
+
+
+# CONFIG_JSON = "config/simulation-charmander.json"
+# CONFIG_JSON = "config/simulation-charmeleon.json"
+# CONFIG_JSON = "config/simulation-charizard.json"
+# CONFIG_JSON = "config/simulation-charmeleon-temporal.json"
+CONFIG_JSON = "config/debugging-measurement-times.json"
+
+###################################################
+
+
+
+
+# constants
+with open(CONFIG_JSON, "r") as file:
+    CONFIG = json.load(file)
```

Comment:
Probably a good idea to add an assert here to make it clear that this needs the temporal data. For example,

```python
with open(CONFIG_JSON, "r") as file:
    CONFIG = json.load(file)
assert CONFIG['simulation_hyperparameters']['report_temporal_data'], "This script is only for configurations that report temporal data."
```

Reply:
Added